content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -86,12 +86,16 @@ is growing every week. Our [Discord channel](http://dvc.org/chat) has almost two
 thousand users. Hundreds more connect with us through email and Twitter. To
 everyone willing to try out DVC, thank you for the opportunity.
 
-## DVC 1.0. is the result of 3 years of learning
+## DVC 1.0 is the result of 3 years of learning
 
 All these contributions, big and small, have a collective impact on DVC's
 development. I'm happy (and a bit nervous) to announce that a pre-release of a
 brand new DVC 1.0 is ready for public beta testing.
 
+```dvc
```

Comment:
You can add: 
`The installation instruction is on [our website](https://dvc.org/doc/install/pre-release) or you can install from pip:`

Reply:
````
You can install the pre-release version from the development branch (instruction on [our website](https://dvc.org/doc/install/pre-release)) or you can from pip:
```

Reply:
Fixed. Accidentally commited directly without creating a PR :(